public_html/wp-content/mu-plugins/jetpack-tweaks/miscellaneous.php

```diff
@@ -127,3 +127,40 @@ function disable_jetpack_blog_follow_emails() {
 	}
 }
 add_filter( 'admin_init', __NAMESPACE__ . '\disable_jetpack_blog_follow_emails' );
+
+/**
+ * Disable Jetpack's automatic spam deletion if the WordCamp is in future or has ended less than month ago.
+ *
+ * Jetpack deletes normally spam submissions after 15 days. Sometimes there are false positivies and
+ * organisers do miss important messages because those get deleted before team manually checks spam folder.
+ * Keep the spam submissions until month has passed from the start of WordCamp, just in case of something
+ * is needed from submissions after WordCamp has ended.
+ */
+function disable_jetpack_spam_delete() {
+	$wordcamp = get_wordcamp_post();
+	$wc_start = $wordcamp->meta['Start Date (YYYY-mm-dd)'][0];
```

Comment:
I think this would trigger an undefined index notice if the start date has not been set yet. One way to avoid this would be to use the null coalescing operator to set an empty value if it doesn't exist.

```suggestion
	$wc_start = $wordcamp->meta['Start Date (YYYY-mm-dd)'][0] ?? '';
```